assets/Mods/shared/Types.lua

```diff
@@ -1188,13 +1191,16 @@ function UDataTable:GetRowStruct() end
 function UDataTable:GetRowMap() end
 
 ---Finds a row in the DataTable by name
+---Returns a UScriptStruct wrapper that provides reference-based access to the row
```

Comment:
```suggestion
---Returns a UScriptStruct that provides reference-based access to the row
```
The wrapper is supposed to be transparent to the modder, so we shouldn't mention the internals, it will only cause confusion.